packages/video_player/example/lib/main.dart

```diff
@@ -386,11 +386,11 @@ void main() {
                     Container(
                       padding: const EdgeInsets.only(top: 20.0),
                     ),
-                    const Text('With remote m3u8'),
+                    const Text('With remote mp4'),
```

Comment:
why did we change this and the url below?

Reply:
that URL was not accessible in public network. was it an internal URL? 
i tried with the URL on the docs and updated it. i can revert it back

Reply:
reverted back to m3u8